packages/account/src/Components/personal-details/__tests__/personal-details.spec.tsx

```diff
@@ -856,41 +883,46 @@ describe('<PersonalDetails/>', () => {
     });
 
     it('should validate idv values along with additional document number when a document type is selected', async () => {
-        shouldShowIdentityInformation.mockReturnValue(true);
+        (shouldShowIdentityInformation as jest.Mock).mockReturnValue(true);
 
         const new_document_data = {
             ...idv_document_data,
-            document_type: { ...idv_document_data.document_type, additional: '12345' },
+            document_type: {
+                ...idv_document_data.document_type,
+                additional: {
+                    display_name: '12345',
+                },
+            },
         };
 
         const new_props = {
-            ...props,
+            ...mock_props,
             value: {
-                ...props.value,
+                ...mock_props.value,
                 ...new_document_data,
             },
             residence_list: default_residence_details,
         };
-        renderwithRouter(<PersonalDetails {...new_props} />);
+        renderwithRouter({ props: new_props });
 
         await waitFor(() => {
             expect(isAdditionalDocumentValid).toHaveBeenCalled();
         });
     });
 
-    it('should disable tax_residence field if it is immutable from BE', () => {
-        isMobile.mockReturnValue(false);
-        isDesktop.mockReturnValue(true);
+    it('should disable last_name field if it is immutable from BE', () => {
```

Comment:
why did we change checking of tax _residence fields

Reply:
Reverted the changes.